### ISSUE TITLE
Data must be string or a buffer error when used with Docker

### ISSUE THREAD

**Issue by techwizrd** [js-cli/js-v8flags#34]
317b166889313b430e3c0c1a83a1fe432f39018f introduced a bug on L14.

`babel-cli` uses this, and when run outside a Docker container, there are no issues. However, when run in a Docker container, it causes it to fail. See the following:

```
2017-04-18T21:34:13.275287181Z crypto.js:74
2017-04-18T21:34:13.275330167Z   this._handle.update(data, encoding);
2017-04-18T21:34:13.275333714Z                ^
2017-04-18T21:34:13.275336783Z 
2017-04-18T21:34:13.275339442Z TypeError: Data must be a string or a buffer
2017-04-18T21:34:13.275342154Z     at Hash.update (crypto.js:74:16)
2017-04-18T21:34:13.275344873Z     at Object.<anonymous> (/app/node_modules/v8flags/index.js:14:81)
2017-04-18T21:34:13.275348204Z     at Module._compile (module.js:571:32)
2017-04-18T21:34:13.275351214Z     at Object.Module._extensions..js (module.js:580:10)
2017-04-18T21:34:13.275353766Z     at Module.load (module.js:488:32)
2017-04-18T21:34:13.275356293Z     at tryModuleLoad (module.js:447:12)
2017-04-18T21:34:13.275358778Z     at Function.Module._load (module.js:439:3)
2017-04-18T21:34:13.275361353Z     at Module.require (module.js:498:17)
2017-04-18T21:34:13.275363884Z     at require (internal/module.js:20:19)
2017-04-18T21:34:13.275367458Z     at Object.<anonymous> (/app/node_modules/babel-cli/lib/babel-node.js:3:18)
```

**Comment by skreis** [comment]
Bumping into the same...

**Comment by andrewpushspring** [comment]
We are seeing the same error running gulp inside of a docker container on Shippable.

**Comment by phated** [comment]
So your users are all undefined? I'll get a patch out shortly, but you might not like it

**Comment by techwizrd** [comment]
Containers do not have defined users as far as I know.

**Comment by phated** [comment]
Published a fix as 2.1.1 that defaults user to an empty string.  @techwizrd thanks for the quick, detailed report.

**Comment by techwizrd** [comment]
You're welcome. Thanks for the quick fix!
